src/Blazor.Gitter.Core/gulpfile.js

```diff
@@ -0,0 +1,20 @@
+var gulp = require("gulp"),
+    sourcemaps = require("gulp-sourcemaps"),
+    sass = require("gulp-sass"),
+    es = require('event-stream');
+
+gulp.task("sass", async function () {
+    var client = gulp.src('content/css/blazor.gitter.scss')
+        .pipe(sourcemaps.init())
+        .pipe(sass())
+        .pipe(sourcemaps.write('.'))
+        .pipe(gulp.dest('../Blazor.Gitter.Client/wwwroot/css'));
```

Comment:
Is it possible to keep the original compiled css name `blazored.gitter.css` ?

Reply:
Yeah, I can probably rename and minify it. Is it OK if I do this tomorrow? It's almost bedtime in my timezone ;-)